.travis/install-operator-sdk.sh

```diff
@@ -1,8 +1,3 @@
 #!/bin/bash
-SDK_VERSION="v0.16.0"
-SDK_RELEASE="https://github.com/operator-framework/operator-sdk/releases/download/${SDK_VERSION}/operator-sdk-${SDK_VERSION}-x86_64-linux-gnu"
-
-echo "install-sdk - installing version ${SDK_VERSION}"
-curl "${SDK_RELEASE}" -Lo $GOPATH/bin/operator-sdk
-chmod +x $GOPATH/bin/operator-sdk
-$GOPATH/bin/operator-sdk version
+bash build/install-operator-sdk.sh && \
```

Comment:
I think that we don't need to specify bash  

```suggestion
build/install-operator-sdk.sh && \
```

Reply:
@liranmauda why not? are we targeting another shell?

Reply:
any reason to not specify bash?

Reply:
@guymguym using shebang line tells that this interpreter for the shell is bash.

Reply:
I know it will work. OK it's a super duper minor point. I bet each of us will change his mind about which style is better. My argument for now is that we don't want to allow various shell support in our code base, and this code style supports this fact.